### PR TITLE
Feature/smithsonian

### PR DESCRIPTION
**Pull request by moltude**
Add Smithsonian XML file harvester
Add Smithsonian mapping & tests 
Update string type -> DcmiType mapping with SI specific terminology 
Add SI to ProviderProfiles and ProviderRegistry

### CODE REVIEW

File: src/main/scala/dpla/ingestion3/enrichments/TypeEnrichment.scala [hunk 1]
```diff
@@ -15,7 +15,6 @@ class TypeEnrichment extends VocabEnrichment[String] {
   val dcmiType = DCMIType()
 
   val DcmiTypeMap: Map[String, IRI] = Map(
-    "appliance" -> dcmiType.Image,
```
Comment (mdellabitta) [inline src/main/scala/dpla/ingestion3/enrichments/TypeEnrichment.scala]: Umm, what about my collection of washer/dryer photos?

Comment (AudreyAltman) [reply src/main/scala/dpla/ingestion3/enrichments/TypeEnrichment.scala]: @mdellabitta and the dad jokes strike again :)

Comment (mdellabitta) [reply src/main/scala/dpla/ingestion3/enrichments/TypeEnrichment.scala]: Damn straight!

Comment (moltude) [reply src/main/scala/dpla/ingestion3/enrichments/TypeEnrichment.scala]: laugh but I spent more time with this book in my public history class than I care to remember 
`https://www.amazon.com/Save-Womens-Lives-History-Machines/dp/0972971009`

**Pull request merged**